src/components/app/ProfileSharing.js

```diff
@@ -62,6 +64,16 @@ UploadingStatus.propTypes = {
 };
 
 class ProfileSharingCompositeButton extends PureComponent {
+  _attemptToShare: Function;
+  _onPermalinkPanelOpen: Function;
+  _onPermalinkPanelClose: Function;
+  _permalinkButtonCreated: Function;
+  _permalinkButton: Object;
+  _uploadErrorButtonCreated: Function;
+  _uploadErrorButton: Object;
+  _permalinkTextFieldCreated: Function;
+  _permalinkTextField: Object;
```

Comment:
1) corrected
2) I'm not sure I understand the desired syntax here.  I'm imagining something like
```
this._permalinkButtonCreated = elem): ButtonWithPanel => void {
      this._permalinkButton = elem;
    };
```
but this is clearly not correct.
3) I could not find these type definitions.  Should I create them or do they exist somewhere already?